endpoint/crypto.go

```diff
@@ -63,13 +68,18 @@ func EncryptText(text string, aesKey []byte, nonceEncoded []byte) (string, error
 	}
 
 	cipherData := gcm.Seal(nonce, nonce, data, nil)
-	return base64.StdEncoding.EncodeToString(cipherData), nil
+	return base64.URLEncoding.EncodeToString(cipherData), nil
 }
 
 // DecryptText decrypt gziped data using a supplied AES encryption key ang ungzip it
 // in case of decryption failed, will return original input and decryption error
 func DecryptText(text string, aesKey []byte) (decryptResult string, encryptNonce string, err error) {
-	block, err := aes.NewCipher(aesKey)
+	decodedaesKey, err := base64.URLEncoding.DecodeString(string(aesKey))
+	if err != nil {
+		fmt.Println("Error decoding base64 when decrypt text:", err)
+		return "", "", err
```

Comment:
This will break users that are already using txt encryption as far as I understand.
Maybe we should provide a migration path, so for example if it fails to decrypt we use the former StdEncoding.